rust/lance-core/src/io/writer/statistics.rs

```diff
@@ -150,13 +468,211 @@ fn get_boolean_statistics(arrays: &[&ArrayRef]) -> StatisticsRow {
     }
 
     StatisticsRow {
-        null_count,
+        null_count: ScalarValue::Int64(Some(null_count)),
         min_value: ScalarValue::Boolean(Some(true_present && !false_present)),
         max_value: ScalarValue::Boolean(Some(true_present || !false_present)),
     }
 }
+fn get_list_statistics(arrays: &[&ArrayRef]) -> StatisticsRow {
```

Comment:
Could we defer list statistics to a future PR? Part of that PR needs to be defining the semantics in the format docs: https://lancedb.github.io/lance/format.html#statistic-values

Reply:
Will comment it out for now.

Reply:
#1486